### PR TITLE
 Attest artifacts

### PR DESCRIPTION
**Pull request by martincostello**
Attest the binaries and packages from the build artifacts.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -77,6 +77,20 @@ jobs:
...
+      if: |
+        runner.os == 'Windows' &&
+        github.event.repository.fork == false &&
+        (github.ref_name == github.event.repository.default_branch ||
+         startsWith(github.ref, 'refs/heads/dev') ||
+         startsWith(github.ref, 'refs/heads/rel/') ||
+         startsWith(github.ref, 'refs/tags/'))
+      with:
+        subject-path: |
+          ./artifacts/bin/AspNet.Security.OAuth.*/Release/**/AspNet.Security.OAuth.*.dll
```
Comment (kevinchalet) [inline .github/workflows/build.yml:91]: Do we need to be ultra specific here or a more generic filter like `./artifacts/bin/**/Release/**/*.dll` could work too?

Comment (martincostello) [reply .github/workflows/build.yml:91]: This avoids attesting anything that's a dependency that we didn't build ourselves and avoids the sample site too.

Comment (kevinchalet) [reply .github/workflows/build.yml:91]: Makes sense 👍🏻

**Comment by martincostello** [comment]
I just discovered the hard way that there's a 64 file limit on attestations, so even if we only did the NuGet packages, that's still to many 😓

**Pull request closed without merging**